src/main/java/software/amazon/cloudformation/LambdaWrapper.java

```diff
@@ -464,6 +471,9 @@ protected void validateModel(final JSONObject modelObject) throws ValidationExce
     private void publishExceptionMetric(final Action action, final Throwable ex, final HandlerErrorCode handlerErrorCode) {
         if (this.metricsPublisherProxy != null) {
             this.metricsPublisherProxy.publishExceptionMetric(Instant.now(), action, ex, handlerErrorCode);
+            this.metricsPublisherProxy.publishExceptionByErrorCodeMetric(Instant.now(), action, handlerErrorCode);
+            // always puts failed events
+            this.metricsPublisherProxy.publishExceptionCountMetric(Instant.now(), action, true);
```

Comment:
I guess this would require a larger refactor, but ideally a 0/1 metric should be emitted in a single place, because we want to be sure that sum(data points with value = 0) + sum(data point with value = 1) = (number of calls to monitored method). (because then the average of that metric represents the error rate)

Emitting false in one place and true in another makes hard to guarantee that, for a single call to the monitored method, exactly one data point is emitted.

Reply:
this is the method for any exception thrown, so it handles any failures as `1`; otherwise when response is successful and not equal to `FAILED` status it emits `0`